code/HabitTracker/app/src/main/java/com/example/habittracker/Habit.java

```diff
@@ -136,7 +169,7 @@ public void addToDB() {
         //  prolly database manager. check w Zarif.
 
         // the attribute names as specified in the schema and the values that correspond
-        String [] attributes = {"reason", "dateStarted", "whatDays", "progress", "display"};
+        String [] attributes = {"reason", "dateStarted", "whatDays", "progressBar", "display"};
```

Comment:
Is there a reason this is "progress bar" and not progress?

Reply:
I have no idea how this line of the code got changed because I didn't even touch that function. I'll change to progress.

Reply:
done